### PR TITLE
GitHub: Improved bug report template

### PR DESCRIPTION
**Pull request by 0x5bfa**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue...

**Validation**
How did you test these changes?
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -18,32 +22,52 @@ body:
...
+        - Windows 10
+        - Windows 10 Release Preview
+        - Windows 10 Beta
+        - Windows 10 Dev
+        - Windows 10 Canary
+        - Windows 11
+        - Windows 11 Release Preview
+        - Windows 11 Beta
+        - Windows 11 Dev
+        - Windows 11 Canary
```
Comment (yaira2) [inline .github/ISSUE_TEMPLATE/bug_report.yml:63]: We need build numbers

Comment (0x5bfa) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: fixed

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -18,32 +22,51 @@ body:
...
       required: true
+
+  # Requirements
   - type: textarea
     attributes:
       label: Requirements
       description: describe the expected behavior (eg. Fix crash that occurs when...). If left blank, the issue will likely be closed without discussion.
-      value: |
-        - eg. Fix crash that occurs when...
+      placeholder: |
```
Comment (yaira2) [inline .github/ISSUE_TEMPLATE/bug_report.yml:31]: Let's keep this as value, I often use it as a starting point when creating bug reports.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**